src/main/java/com/iridium/iridiumskyblock/listeners/BlockBreakListener.java

```diff
@@ -66,4 +68,21 @@ public void monitorBlockBreak(BlockBreakEvent event) {
         });
     }
 
+    @EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
+    public void onHangingBreakByEntityEvent(HangingBreakByEntityEvent event) {
+        if (event.getRemover() instanceof Player) {
```

Comment:
users can throw egg-snowball to itemframes so u also need check this for projectiles

Reply:
Even arrows, if I remember correctly